internal/service/securityhub/configuration_policy.go

```diff
@@ -0,0 +1,622 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package securityhub
+
+import (
+	"context"
+	"errors"
+	"log"
+	"math"
+
+	"github.com/YakDriver/regexache"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/securityhub"
+	"github.com/aws/aws-sdk-go-v2/service/securityhub/types"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/enum"
+	"github.com/hashicorp/terraform-provider-aws/internal/errs/sdkdiag"
+	"github.com/hashicorp/terraform-provider-aws/internal/tfresource"
+	"github.com/hashicorp/terraform-provider-aws/internal/verify"
+)
+
+// @SDKResource("aws_securityhub_configuration_policy")
+func ResourceConfigurationPolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateWithoutTimeout: resourceConfigurationPolicyCreate,
+		ReadWithoutTimeout:   resourceConfigurationPolicyRead,
+		UpdateWithoutTimeout: resourceConfigurationPolicyUpdate,
+		DeleteWithoutTimeout: resourceConfigurationPolicyDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringMatch(
+					regexache.MustCompile(`[A-Za-z0-9\-\.!*/]+`),
+					"Only alphanumeric characters and the following ASCII characters are permitted: -, ., !, *, /",
+				),
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"security_hub_policy": {
```

Comment:
I had re-named this `policy_member` during my lint refactor but got undone in a few files (apologies again). 

Acceptance tests still reference new name in most places so we will need to update everything to be consistent. I'll leave it up to @ewbankkit to decide on proper name.

I can re-run the multi-account acceptance tests again once everything is fixed.